modules/capture-video/src/main/native/linux/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.10)
+project(capturevideo)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ../../../../../build/native/linux)
+include_directories(/opt/java/include)
+include_directories(/opt/java/include/linux)
+
+add_library(CaptureVideo SHARED
+    ../src/capture.c
+)
+target_link_libraries(CaptureVideo v4l2 udev)
```

Comment:
This is needed:
```
sudo apt-get install libudev-dev
sudo apt-get install libv4l-dev
```
not sure if as part of CMakeList

Reply:
that should be in the prerequisites -- I doubt this can be done via cmake